README.md

```diff
@@ -105,6 +105,20 @@ This command runs all available unit tests.
 
 ## Experimental features
 
+### `pdk console` command
```

Comment:
@jbondpdx Can you give these docs a quick look.

* `To use just execute` avoid `just` and `easy`

* ``~/modules/ntp  $ ... ` Avoid shell specific things e.g. a PowerShell user would see this prefixed differently.

Reply:
I update the README with the fixes and also added some extra bits I thought were important.

Reply:
Sorry I didn't see this sooner, @glennsarti , @logicminds ... I think it got lost in a mass of GitHub emails. Thank you for both for these docs!